### DISCUSSION TITLE
KextViewer

### DISCUSSION THREAD

**Discussion by avincent98144** [community/community#39181] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Just noticed KextViewer is no longer populating anything in the window, completely empty!  A 13.2 beta 2 bug?

**Comment by david-engelmann** [comment]
This discussion section might not yield the best responses for your question. The community/community discussion is mainly for issues related to Github itself. You'll be assisted better in a KextViewer specific forum

**Comment by avincent98144** [comment]
.. oh shoot .. thought I was in that particular forum ;-)

**Comment by david-engelmann** [comment, marked as answer]
@avincent98144 No problem! Don't forget to mark as an answer!
